web/app.go

```diff
@@ -174,10 +173,10 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 	webEngine.GET("/sapsystems/:id", NewSAPResourceHandler(deps.hostsService, deps.sapSystemsService))
 	webEngine.GET("/databases", NewHanaDatabaseListHandler(deps.consul, deps.hostsService, deps.sapSystemsService, deps.tagsService))
 	webEngine.GET("/databases/:id", NewSAPResourceHandler(deps.hostsService, deps.sapSystemsService))
-	webEngine.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
 
 	apiGroup := webEngine.Group("/api")
 	{
+		webEngine.GET("/docs/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
 		apiGroup.GET("/ping", ApiPingHandler)
```

Comment:
even this is moved in the block, the route is still under `webEngine`. Was this the intention or we should move it under `apiGroup`?

Reply:
nope, that's an error, thanks for catching it :D